nixos/modules/security/apparmor/profiles.nix

```diff
@@ -0,0 +1,11 @@
+{ config, lib, pkgs, ... }:
+let apparmor = config.security.apparmor; in
+{
+config.security.apparmor.packages = [ pkgs.apparmor-profiles ];
+config.security.apparmor.policies."bin.ping".profile = lib.mkIf apparmor.policies."bin.ping".enable ''
+  include "${pkgs.iputils.apparmor}/bin.ping"
```

Comment:
Does this work with our ping that lives in `/run/wrappers`?

Reply:
Yes, but the relevant code is where `ping`'s wrappers are added, in `nixos/modules/tasks/network-interfaces.nix`: https://github.com/NixOS/nixpkgs/pull/101071/files#diff-0880fae3bbdeb321ece213e5fdb68064fc7eade20cdcfd27b14268e811e36d77